src/apps/filters/h3.c

```diff
@@ -117,7 +161,8 @@ H3Error latLngToCellCmd(int argc, char *argv[]) {
     return e;
 }
 
-H3Error cellToBoundaryCmd(int argc, char *argv[]) {
+SUBCOMMAND(cellToBoundary,
+           "Convert an H3 cell to a WKT POLYGON defining its boundary") {
```

Comment:
Same, maybe require a flag even for the default output format to give us more room to maneuver in the future? Plus it makes the output format really explicit...